addon/components/o-s-s/information-section.ts

```diff
@@ -0,0 +1,16 @@
+import Component from '@glimmer/component';
+import type { IconNames } from './iconName.enum';
+
+interface InformationSectionArgs {
+  badgeIcon?: keyof typeof IconNames;
+  imageUrl?: string;
+  title: string;
+  subtitle?: string;
+  plain?: boolean;
+}
+
+export default class OSSInformationSection extends Component<InformationSectionArgs> {
+  get plainClass(): string {
+    return this.args.plain === false ? 'background-color-gray-50' : 'background-color-white';
```

Comment:
For `undefined` value, you want the plain effect? Like the `true` value?

Reply:
Yep, thats it, plain === true would mean the header & content have a white background, plain === false would only have the header in gray. plain === undefined defaults out to all white